php/values.yaml

```diff
@@ -131,6 +131,7 @@ nginx:
   lifecycle:
     postStart: []
     # postStart: ["/bin/sh", "-c", "echo Hello > /tmp/message"]
+    preStop: []
```

Comment:
In my opinion,  it would be better to set a default value to `nginx.lifecycle.preStop` for safety shutdown.

```
preStop: ["/bin/sh", "-c", "sleep {{ .Values.fpm.processControlTimeout | default 0 }}; nginx -s quit; sleep 5"]
```